### PR TITLE
fixed offset option

### LINKED ISSUE #465: Overlay module offset parameter doesn't work
The Overlay module has a parameter called "offset" which lets you set which step is overlaid onto. But it doesn't seem to work:

See it malfunctioning here in the final step:

http://sequencer.publiclab.org/examples/#steps=crop{x:0|y:0|w:611|h:479},gradient{},colormap{colormap:stretched},crop{x:0|y:0|w:611|h:60},resize{resize:25%25},overlay{x:5|y:5|offset:-5}

Compare to:

http://sequencer.publiclab.org/examples/#steps=crop{x:0|y:0|w:611|h:479},gradient{},colormap{colormap:stretched},crop{x:0|y:0|w:611|h:60},resize{resize:25%25},overlay{x:5|y:5|offset:-3}

It always seems to source the same image!

The code for the Overlay module is here: https://github.com/publiclab/image-sequencer/tree/master/src/modules/Overlay

### PR DESCRIPTION
**Pull request by JKusio**
Fixes #465

Fixed the offset option from overlay module.

Sometimes the offset value is a string, and it need to be parsed into integer.

### CODE REVIEW

**Comment by jywarren** [comment]
Oh, great catch. Would you mind compiling this with `grunt build` and checking in the dist folder? Thanks a lot!!

**Comment by KusioDev** [comment]
Compiled, and the code I wrote is there and it's working 👍

**Comment by jywarren** [comment]
Awesome, can you push up your additions? Thanks!!

**Comment by KusioDev** [comment]
Push the file from dist folder?

**Comment by jywarren** [comment]
yes, that's right! thank you, i didn't see it in the files in this PR 👍

**Comment by jywarren** [comment]
Hmm... sorry, sometimes this happens, can you rebase your dist files over the latest? I think because I just merged a feature from @Mridul97 -- a Contrast module! Apologies for the trouble and I'm happy to do the rebase myself if it's too much trouble. We're just doing really fast work on this project and things are happening one after the other! 🙌

**Comment by jywarren** [comment]
you may need to regenerate them. again, i'm happy to help!

**Comment by KusioDev** [comment]
I've deleted the last commit - learning something new!

**Comment by jywarren** [comment]
Aha, excellent! Now i think if you could rebase this on top of the latest main branch and recompile, you can check in the `dist` files one more time and it'll be based on the latest main branch, with no conflicts. 

If it's helpful, there's some guidance here: 

> You can do that by following the process outlined here, but you may be able to skip step 2 if your master is still in sync with publiclab's master: https://publiclab.org/wiki/contributing-to-public-lab-software#Rewinding+the+master+branch

And at https://publiclab.org/w/developers#Resources

Thanks!

**Comment by jywarren** [comment]
Hi, @KusioDev did you want me to help out here? Thanks a lot!

**Comment by tech4GT** [comment]
Okay this looks great @KusioDev !! :smile: 
@jywarren Should I build and rebase this?

**Comment by jywarren** [comment]
Sure, that'd be great. @KusioDev have you read through how to compile the code in /src/ into /dist/ folder? Thanks, all!

**Comment by gauravano** [comment]
@KusioDev  it's near completion so just checking, in case you need help. Thanks!

**Comment by HarshKhandeparkar** [comment]
@KusioDev this PR is nearly ready to be merged. Can you please fix the conflicts? Comment below if you need any help. Thanks! 😊

**Comment by HarshKhandeparkar** [comment]
@JKusio please rebase your branch. Ask if you need any help. Thanks!

**Comment by Harshithpabbati** [comment]
@jywarren @jkusio is not active from many days. So shall I rebase it???

**Comment by HarshKhandeparkar** [comment]
Sure.

**Comment by Harshithpabbati** [comment]
@jywarren Done :)

File: src/modules/Overlay/Module.js [hunk 1]
```diff
@@ -1,21 +1,29 @@
 module.exports = function Dynamic(options, UI, util) {
 
+<<<<<<< HEAD
```
Comment (HarshKhandeparkar) [inline src/modules/Overlay/Module.js]: Please remove thw conflicts

**Comment by HarshKhandeparkar** [comment]
@Harshithpabbati

**Comment by Harshithpabbati** [comment]
Yeah Doing it.

File: src/modules/Overlay/Module.js [hunk 2]
```diff
@@ -3,19 +3,22 @@ module.exports = function Dynamic(options, UI, util) {
     var defaults = require('./../../util/getDefaults.js')(require('./info.json'));
     options.x = options.x || defaults.x;
     options.y = options.y || defaults.y;
+    options.offset = options.offset || -2;
```
Comment (HarshKhandeparkar) [inline src/modules/Overlay/Module.js]: Can you also load this option from the defaults?

File: package-lock.json [hunk 3]
```diff
@@ -3420,13 +3437,15 @@
...
-          "dev": true
+          "dev": true,
+          "optional": true
         },
         "yallist": {
           "version": "3.0.2",
           "resolved": "https://registry.npmjs.org/yallist/-/yallist-3.0.2.tgz",
           "integrity": "sha1-hFK0u36Dx8GI2AQcGoN8dz1ti7k=",
-          "dev": true
+          "dev": true,
```
Comment (jywarren) [inline package-lock.json]: Did we want all of these, @HarshKhandeparkar @Harshithpabbati ? Just checking!

Comment (HarshKhandeparkar) [reply package-lock.json]: @jywarren I think you reviewed a commit not the whole changes.

Comment (jywarren) [reply package-lock.json]: Yes, i think the whole thing looks fine, but wasn't sure these package-lock.json changes were intended to be added? Thanks!

**Comment by HarshKhandeparkar** [comment]
Seems like this is an old issue and may be fixed. Stale PR. Closing this. Please feel free to reopen if necessary @JKusio

**Pull request closed without merging**